### PR TITLE
refactor: update helm chart

### PR DESCRIPTION
**Pull request by starnop**
Signed-off-by: Starnop <starnopg@gmail.com>

### CODE REVIEW

File: values.yaml [hunk 1]
```diff
@@ -17,7 +17,7 @@ dfclient:
   ##
   image:
     repository: dragonflyoss/dfclient
-    tag: 0.4.3
+    tag: 1.0.0-alpha
```
Comment (yeya24) [inline values.yaml]: update to 1.0.0

File: values.yaml [hunk 2]
```diff
@@ -91,13 +95,26 @@ dfclient:
...
+    #     - regx: index.docker.io
+
 supernode:
   name: supernode
   ## supernode container image
   ##
   image:
     repository: dragonflyoss/supernode
-    tag: 0.4.3
+    tag: 1.0.0-alpha
```
Comment (yeya24) [inline values.yaml]: ditto

Comment (starnop) [reply values.yaml]: Done.

File: values.yaml [hunk 3]
```diff
@@ -17,7 +17,7 @@ dfclient:
   ##
   image:
     repository: dragonflyoss/dfclient
-    tag: 0.4.3
+    tag: latest
```
Comment (yeya24) [inline values.yaml]: Don' t use `latest`. Use a specific version instead. 1.0.0

File: values.yaml [hunk 4]
```diff
@@ -91,13 +95,26 @@ dfclient:
...
+    #     - regx: index.docker.io
+
 supernode:
   name: supernode
   ## supernode container image
   ##
   image:
     repository: dragonflyoss/supernode
-    tag: 0.4.3
+    tag: latest
```
Comment (yeya24) [inline values.yaml]: Ditto

File: values.yaml [hunk 5]
```diff
@@ -243,4 +274,124 @@ supernode:
...
+      youngGCThreshold: 100G
+
+      #  FullGCThreshold if the available disk space is less than FullGCThreshold
+      #  and the supernode should gc all task files which are not being used.
+      #  default: 5GB
+      fullGCThreshold: 5G
+
+      # IntervalThreshold is the threshold of the interval at which the task file is accessed.
+      # default: 2h0m0s
+      IntervalThreshold: 2h
```
Comment (yeya24) [inline values.yaml:308]: Do we need such detailed config template file here? 
I preferring removing the comments and putting these comments in a separate config doc.

Comment (starnop) [reply values.yaml:308]: Agree.

Comment (starnop) [reply values.yaml:308]: Done.

File: values.yaml [hunk 6]
```diff
@@ -29,15 +29,15 @@ dfclient:
   args: []
 
   ## Node tolerations for dfclient scheduling to nodes with taints
-  ## Ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/
+  ## Ref: https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/
   ##
   tolerations: 
     - key: "node-role.kubernetes.io/master"
       operator: "Exists"
       effect: "NoSchedule"
```
Comment (yeya24) [inline values.yaml:37]: Do we need to add more default tolerations? See this doc https://github.com/dragonflyoss/website/blob/master/blog/zh-cn/deploy-dragonfly-on-k8s.md

File: values.yaml [hunk 7]
```diff
@@ -243,4 +275,37 @@ supernode:
...
+      homeDir: /home/admin/supernode
+      schedulerCorePoolSize: 10
+      peerUpLimit: 5
+      peerDownLimit: 4
+      eliminationLimit: 5
+      failureCountLimit: 5
+      systemReservedBandwidth: 20M
+      maxBandwidth: 200M
+      enableProfiler: false
+      debug: false
```
Comment (yeya24) [inline values.yaml:299]: Is it better to enable the debug configuration? Only debug is enabled, can dfget uploads its metrics to supernode. Then users can check the log file.

File: values.yaml [hunk 8]
```diff
@@ -77,9 +77,13 @@ dfclient:
   ## Additional dfclient hostPath mounts
   ##
   extraHostPathMounts: []
-    # - name: logs
-    #   mountPath: /root/.small-dragonfly/logs
-    #   hostPath: /var/dragonfly/logs
```
Comment (yeya24) [inline values.yaml:82]: For me, this hostpath volume can help collect these dfclient logs by log agent. Is it better to keep it?

**Comment by yeya24** [comment]
Other changes LGTM. We just need to discuss the rest of my comments

File: values.yaml (hunk 7 above)
Comment (starnop) [reply values.yaml:299]: I don't think so. The supernode with the debug model will output a lot of logs. 

> Only debug is enabled, can dfget uploads its metrics to supernode.

Does that mean `dfget will not upload its metrics to supernode` with debug false?
If that, I think it's unreasonable and we should fix it.

File: values.yaml (hunk 6 above)
Comment (starnop) [reply values.yaml:37]: I don't think it's necessary. 😄

File: values.yaml (hunk 8 above)
Comment (starnop) [reply values.yaml:82]: Agree. I will change that.

Comment (starnop) [reply values.yaml:82]: Done.

**Comment by starnop** [comment]
@yeya24 PTAL again. THX.

File: values.yaml (hunk 7 above)
Comment (yeya24) [reply values.yaml:299]: Oh, this is my miss. Dfget still can upload metrics to Supernode. But supernode itself has another log file `dfclient.log`, which contains these metrics in logs. The log level is debug so only debug level can see these logs.
But metrics uploading still works when debug is false

Comment (yeya24) [reply values.yaml:299]: If you think debug: false is better, I will merge this.

**Review by yeya24 — APPROVED** [review]
LGTM

**Pull request merged**